assets/js/blocks/product-categories/style.scss

```diff
@@ -67,11 +67,11 @@
 	border: none;
 	cursor: pointer;
 	background: none;
-	padding: 0 rem(8, 13);
+	padding: 0 0.5em;
```

Comment:
curious why you're not using `em()` for all these values?

Reply:
Like before, I think using `em()` (or `rem()` as it was until now) makes this code harder to understand.

In fact, there is a curious bug here. That element had 13px font-size, and we needed a padding of 8px, so it was `rem(value: 8, base: 13)`. But in #2498 I removed the font-size 13px but forgot to update this mixin. :man_facepalming: So instead of 8px now the padding is 9.8px (=8/13*16).

To avoid the need to do all this kind of calculations, I think we should get used to use `em` values when possible.